main.go

```diff
@@ -35,7 +35,7 @@ func main() {
 		os.Exit(0)
 	}
 	log.Println("Scraping metrics from", fmt.Sprintf("%s:%d", cfg.SquidHostname, cfg.SquidPort))
-	e := collector.New(cfg.SquidHostname, cfg.SquidPort, cfg.Login, cfg.Password, cfg.Labels)
+	e := collector.New(cfg.SquidHostname, cfg.SquidPort, cfg.Login, cfg.Password, cfg.Labels, cfg.ServiceTimes)
```

Comment:
See the comments above, then you can call `c.ExtractServiceTimes = cfg.ExtractServiceTimes`.